server/sonar-server/src/main/java/org/sonar/server/project/ws/SearchAction.java

```diff
@@ -130,7 +131,7 @@ private static SearchWsRequest toSearchWsRequest(Request request) {
       .setPage(request.mandatoryParamAsInt(Param.PAGE))
       .setPageSize(request.mandatoryParamAsInt(Param.PAGE_SIZE))
       .setVisibility(request.param(PARAM_VISIBILITY))
-      .setAnalyzedBefore(request.paramAsDateTime(PARAM_ANALYZED_BEFORE))
+      .setAnalyzedBefore(request.param(PARAM_ANALYZED_BEFORE))
```

Comment:
I would have done such update in the commit that as added the parameter => it allows to understand better which update is part of which commit, when investigating history

Reply:
I'll merge the 2 commits anyway but indeed, you're right